### PR TITLE
Ui controls

### LINKED ISSUE #62: Enable resize/zoom on the PDF
In the standard PDF viewer in JupyterLab (at least in Chrome, my only available browser) one can easily zoom in and out on the PDF. This is helpful on smaller screens such as ultrabooks since one can zoom in enough to get rid of the margins and just be looking at the text area. Right now, this extension does not support the same, so one always sees the entire page of the PDF. 

It would be tremendously useful if one could still zoom in and out of the PDF, either by using the standard PDF viewer (perhaps with modifications) or adding the feature directly into this packages existing PDF viewer.

### PR DESCRIPTION
**Pull request by ian-r-rose**
Adds a minimal toolbar to the PDFJS viewer, allowing the user to zoom and navigate. Fixes #62

### CODE REVIEW

**Comment by ian-r-rose** [comment]
Okay, this is ready for a look.

**Comment by mpacer** [comment]
So this takes care of the nav bar disappearing as well as some of the double load.

Does it handle the "there is no synctex file and you just triggered synctex" case?

**Comment by ian-r-rose** [comment]
I am not sure. I wasn't able to reproduce that problem last night, but I'd appreciate if you poke it a bit.

File: src/index.ts [hunk 1]
```diff
@@ -458,7 +453,8 @@ function addSynctexCommands(app: JupyterLab, editorTracker: IEditorTracker, pdfT
       let widget = editorTracker.currentWidget;
       if (widget) {
         // Get the cursor position.
```
Comment (mpacer) [inline src/index.ts]: I'd prefer a comment to exist explaining the synctex indexing vs. the editor's usual indexing.

File: src/index.ts [hunk 2]
```diff
@@ -478,7 +473,7 @@ function addSynctexCommands(app: JupyterLab, editorTracker: IEditorTracker, pdfT
             return;
           }
           // Scroll the pdf.
-          pdfWidget.setScroll(edit);
+          pdfWidget.position = { ...edit, x: 0 };
```
Comment (mpacer) [inline src/index.ts]: why is x always being set to 0?

File: src/pdf.ts [hunk 3]
```diff
@@ -119,8 +147,22 @@ class PDFJSViewer extends Widget implements DocumentRegistry.IReadyWidget {
...
+    // Clamp the page number.
+    const pageNumber = Math.max(
+      Math.min(pos.page, this._pdfViewer.pagesCount + 1), 1);
+    // Scroll page into view using a very undocumented
+    // set of options.
+    this._pdfViewer.scrollPageIntoView({
+      pageNumber,
+      destArray: [
+        pageNumber,
+        { name: 'XYZ' },
```
Comment (mpacer) [inline src/pdf.ts]: what is this 'XYZ'?

**Comment by ian-r-rose** [comment]
I just added a fit-to-page button.

**Comment by mpacer** [comment]
Looks good!

**Pull request merged**